docs/reference/src/main/asciidoc/environments.asciidoc

```diff
@@ -174,15 +174,15 @@ No further configuration is needed when starting Jetty as an embedded webapp ser
 However, if you’re using a Jetty standalone instance one more configuration step is required, one of the
 jetty modules listed below needs to be enabled.
 
-===== Jetty `decorate` Module
+===== Jetty `cdi-decorate` Module
 
-Since Jetty-9.4.20 and Weld 3.1.2, the Weld/Jetty integration uses the jetty `decorate` module.
-To activate this module in jetty the argument `--module=decorate` needs to be added to the
+Since Jetty-9.4.20 and Weld 3.1.2, the Weld/Jetty integration uses the jetty `cdi-decorate` module.
```

Comment:
Maybe we should rather say `Weld 3.1.2.Final and newer` to indicate the doc is valid even for future versions until stated otherwise.

Reply:
I can add the `Final` but I can't word smith anything that works well with `and newer` as there are too many `and`s
```
For Jetty-9.4.20 and newer and Weld 3.1.2.Final and newer...
```
I think the `Since` covers the `and newer`:

```
Since Jetty-9.4.20 and Weld 3.1.2.Final...
```